src/services/inlayHints.ts

```diff
@@ -422,6 +450,307 @@ export function provideInlayHints(context: InlayHintsContext): InlayHint[] {
+                    }
+                    parts.push({ text: ": " });
+                    if (mappedType.type) {
+                        visitDisplayPart(mappedType.type);
+                    }
+                    parts.push({ text: "; }" });
+                    break;
+                case SyntaxKind.LiteralType:
+                    visitDisplayPart((node as LiteralTypeNode).literal);
+                    break;
+                case SyntaxKind.FunctionType:
+                    const functionType = node as FunctionTypeNode;
+                    if (functionType.typeParameters) {
+                        parts.push({ text: "<" });
+                        visitDisplayPartList(functionType.typeParameters, ", ");
+                        parts.push({ text: ">" });
+                    }
+                    parts.push({ text: "(" });
+                    visitDisplayPartList(functionType.parameters, ", ");
+                    parts.push({ text: ")" });
+                    parts.push({ text: " => " });
+                    visitDisplayPart(functionType.type);
+                    break;
+                case SyntaxKind.ImportType:
+                    const importType = node as ImportTypeNode;
+                    if (importType.isTypeOf) {
+                        parts.push({ text: "typeof " });
+                    }
+                    parts.push({ text: "import(" });
+                    visitDisplayPart(importType.argument);
+                    if (importType.assertions) {
+                        parts.push({ text: ", { assert: " });
+                        visitDisplayPartList(importType.assertions.assertClause.elements, ", ");
+                        parts.push({ text: " }" });
+                    }
+                    parts.push({ text: ")" });
+                    if (importType.qualifier) {
+                        parts.push({ text: "." });
+                        visitDisplayPart(importType.qualifier);
+                    }
+                    if (importType.typeArguments) {
+                        parts.push({ text: "<" });
+                        visitDisplayPartList(importType.typeArguments, ", ");
+                        parts.push({ text: ">" });
+                    }
+                    break;
+                case SyntaxKind.PropertySignature:
+                    const propertySignature = node as PropertySignature;
+                    if (propertySignature.modifiers) {
+                        visitDisplayPartList(propertySignature.modifiers, " ");
+                    }
+                    visitDisplayPart(propertySignature.name);
+                    if (propertySignature.questionToken) {
+                        parts.push({ text: "?" });
+                    }
+                    if (propertySignature.type) {
+                        parts.push({ text: ": " });
+                        visitDisplayPart(propertySignature.type);
+                    }
+                    break;
+            }

```

Comment:
Are there any nodes left at the end of this switch case? If not, can we make it exhaustive such that we can `assertNever` or Debug fail if unexpected, such that we'll never forget to add here?